### ISSUE TITLE
swingset-runner test flake

### ISSUE THREAD

**Issue by michaelfig** [Agoric/agoric-sdk#1679]
@FUDCo apparently there's a flakey test in swingset-runner: https://github.com/Agoric/agoric-sdk/pull/1678/checks?check_run_id=1064886156#step:32:18
Can you PTAL at it?

_Originally posted by @michaelfig in https://github.com/Agoric/agoric-sdk/pull/1678#issuecomment-686231987_

**Comment by michaelfig** [comment]
This is apparently germane. I reproduced it once in about 15 tries... there's some race of some kind between the different storages and the on-disk state:

```
UnhandledPromiseRejectionWarning: [Error <Object <[Object: null prototype] {}>>: Bundle bundle-bot not found
  at create (kernel/packages/SwingSet/src/kernel/dynamicVat.js:67:13)
  at recreateVatDynamically (kernel/packages/SwingSet/src/kernel/dynamicVat.js:60:12)
  at Object.start (kernel/packages/SwingSet/src/kernel/kernel.js:842:13)
  at async buildVatController (/Users/michael/agoric/agoric-sdk/packages/SwingSet/src/controller.js:518:27)
  at async main (/Users/michael/agoric/agoric-sdk/packages/swingset-runner/src/main.js:343:22)]
  run encouragmentBot demo with lmdb
```

**Comment by FUDCo** [comment]
Fixed by #1683
